__tests__/app.test.js

```diff
@@ -85,7 +85,16 @@ describe("GET/api/articles/:article_id", () => {
         expect(typeof article.topic).toEqual("string");
         expect(typeof article.created_at).toEqual("string");
         expect(typeof article.votes).toEqual("number");
-        expect(typeof article.article_img_url).toEqual("string");
+        expect(typeof article.article_img_url).toEqual("string")
+        expect(typeof article.comment_count).toEqual("number");
+      });
+  }); test("200: should respond with an article object, containing the correct comment count", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body;
+        expect(article.comment_count).toBe(2)
```

Comment:
I can see you have added this test to the existing endpoint. Ideally, you would want to add a new test block to check that the comment count has the correct keys 👀